src/adapters/MeetingsJSONAdapter.js

```diff
@@ -581,6 +598,64 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return concat(default$, shareEvent$);
   }
 
+  /**
+   * Handles the toggle of the roster.
+   *
+   * @param {string} ID ID of the meeting for which to update display
+   * @private
+   */
+  async handleRoster(ID) {
```

Comment:
to align with the other controls' display function names, let's call this `rosterControl`